js/build.js

```diff
@@ -10,9 +10,14 @@ Fliplet.Widget.instance('chart-pie-1-1-0', function(data) {
   if (themeInstance) {
     Object.assign({}, themeInstance.data.values);
 
+    var themeValue = themeInstance.data.values;
+    var widgetValue = getColors(themeInstance.data.widgetInstances);
+
+    themeValues = Object.assign(themeValue, widgetValue);
+
     _.forEach(themeInstance.data.widgetInstances, function(widgetProp) {
```

Comment:
I realised that `themeInstance.data.widgetInstances` could be quite large because it has all the widget instances across all the screens. Let's shortcut this. Assuming you get the code right, we can skip QA with Kathi to save time.

```js
    _.some(themeInstance.data.widgetInstances, function(widgetProp) {
      if (chartId === widgetProp.id) {
        Object.assign(widgetProp.values, themeValues);

        return true;
      }
    });
```